tensorflow/lite/delegates/gpu/common/model_builder.cc

```diff
@@ -1810,13 +1810,16 @@ class StridedSliceOperationParser : public TFLiteOperationParser {
     if (attr.strides.h < 0 || attr.strides.w < 0 || attr.strides.c < 0) {
```

Comment:
now you're dividing things by attr.strides.h, w, and c.  thus, this condition now needs to include =, i.e.

if (attr.strides.h <= 0 || ...)

Reply:
Good catch. But I would suggest adding a separate check for zero strides. Just so different types of error don't get mixed together i.e. reverse slicing being a valid operation that just isn't supported (yet) and zero strides which are invalid.